arbitrum-client/src/constants.rs

```diff
@@ -53,6 +53,8 @@ pub const SELECTOR_LEN: usize = 4;
 pub const WALLET_UPDATED_EVENT_NAME: &str = "WalletUpdated";
 /// The abi signature for the `NullifierSpent` event
 pub const NULLIFIER_SPENT_EVENT_NAME: &str = "NullifierSpent";
+/// The interval at which to poll for event filters and pending transactions
+pub const BLOCK_POLLING_INTERVAL: u64 = 0;
```

Comment:
Open to pushback on setting this to 0, not sure of the networking drawbacks of querying this quickly.

Reply:
the chain events worker polls using these same parameters [here](https://github.com/renegade-fi/renegade/blob/main/workers/chain-events/src/listener.rs#L86-L96). I believe the stream it constructs will poll under the hood. This will go on continuously so this is probably too frequent for chain events.

Can we expose this as a config option on the client and build a separate client for `chain-events`?

Reply:
and for this one can we do like 100ms or something? not a fan of spin loops like this

Reply:
Sounds good, will add `polling_interval` to the config and instantiate a new client for `chain-events`. 7s is too long for the `chain-events` client as well right?

Reply:
lets set it to 7s for now, the behavior `chain-events` exposes is only important for multi-cluster so we can tune it then